mmdet/models/data_preprocessors/track_data_preprocessor.py

```diff
@@ -0,0 +1,251 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from numbers import Number
+from typing import Dict, List, Optional, Sequence, Union
+
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from mmengine.model import BaseDataPreprocessor
+
+from mmdet.registry import MODELS
+from mmdet.structures import TrackDataSample
+from mmdet.structures.mask import BitmapMasks
+
+
+@MODELS.register_module()
+class TrackDataPreprocessor(BaseDataPreprocessor):
```

Comment:
Can inherit DetDataPreprocessor

Reply:
If inherit DetDataPreprocessor, TrackDataPreprocessor will have some unnecessary property, such as `gt_instances`, and `pred_instances`. This is not in line with the function of `TrackDataPreprocessor`.